tests/lib-injection/dd-lib-python-init-test-django-uvicorn/Dockerfile

```diff
@@ -1,9 +1,9 @@
-FROM python:3.9
+FROM python:3.12
```

Comment:
Why does the python and django version upgrade only happen for this Dockerfile?

Reply:
Answered offline: We needed test coverage for 3.12, and we had two for 3.9, so this was to ensure we had a test case for 3.12 since it would be newly supported.